### PR TITLE
Removes more unused configs

### PR DESCRIPTION
**Pull request by DominikPanic**
The guest stuff is already handled via a config in IsBanned rather than client/New

### CODE REVIEW

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 1]
```diff
@@ -37,7 +37,7 @@
 	if(!istype(user) || !isturf(user.loc) || user.incapacitated())
 		return FALSE
 
-	if(!CONFIG_GET(flag/unlimited_rotate_speed))
```
Comment (spookydonut) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm]: disagree with combining these

File: code/modules/vehicles/multitile/hardpoints.dm [hunk 2]
```diff
@@ -278,7 +278,7 @@ Currently only has the tank hardpoints
 	log_combat(usr, usr, "fired the [src].")
 	log_explosion("[usr] fired the [src] at [AREACOORD(loc)].")
 	P.fire_at(T, owner, src, P.ammo.max_range, P.ammo.shell_speed)
-	if(!CONFIG_GET(flag/tank_mouth_noise))
```
Comment (spookydonut) [inline code/modules/vehicles/multitile/hardpoints.dm]: disagree with combining these, either commit to removing the joke noises or keep it as a separate thing

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm (hunk 1 above)
Comment (DominikPanic) [reply code/game/objects/structures/stool_bed_chair_nest/chairs.dm]: I disagree with this.

File: code/modules/vehicles/multitile/hardpoints.dm (hunk 2 above)
Comment (DominikPanic) [reply code/modules/vehicles/multitile/hardpoints.dm]: I disagree with this.

Comment (Rohesie) [reply code/modules/vehicles/multitile/hardpoints.dm]: To be fair if we consider the possible usage, then individual configs is better, as it gives the admins the choice of what to enable and what not to.

Comment (DominikPanic) [reply code/modules/vehicles/multitile/hardpoints.dm]: It has never been used and I sure hope it never will. This simply doesn't fit our server and having 200 configs for every little thing is not really feasible nor maintainable.

Comment (Rohesie) [reply code/modules/vehicles/multitile/hardpoints.dm]: Eh, remove both then.

Comment (DominikPanic) [reply code/modules/vehicles/multitile/hardpoints.dm]: Done

**Pull request merged**